scraper/scripts/release.sh

```diff
@@ -2,17 +2,15 @@
 
 # Generate a Release Changelog
 # 1. Clone the latest ifsc-api repo
-git clone https://github.com/razorpay/ifsc-api.git --depth 1
+# git clone https://github.com/razorpay/ifsc-api.git --depth 1
```

Comment:
?

Reply:
Was running it offline, had it commented. Thanks :+1: